examples/rust/pipe-ping-pong.rs

```diff
@@ -168,109 +168,164 @@ fn close(libos: &mut LibOS, pipeqd: QDesc) {
     }
 }
 
-//======================================================================================================================
-// server()
-//======================================================================================================================
+// The pipe server.
+pub struct PipeServer {
+    /// Underlying libOS.
+    libos: LibOS,
+    /// Tx pipe qd.
+    tx_pipeqd: Option<QDesc>,
+    /// Rx pipe qd.
+    rx_pipeqd: Option<QDesc>,
```

Comment:
Could you rollback changes applied to the name of these variables?

Reply:
I changed the names to make them easier to read. The difference in the name is at the very beginning now, so they stand out and reduce ambiguity while reading the code. Do let me know if you think differently.

Reply:
I have a different perspective on this. Here is the rationale:

- The code base uses the previous name convention. Checkout mainline code for Catloop.
- This PR does not rename variables repository-wide.
- We end up having different naming conventions for this variable.
- Having the new naming style being easier/harder is something personal.

Reply:
I agree with Pedro. In fact, I would strongly recommend that we get rid of the Option<> in the data structure and just replace it directly with the variable.

Reply:
Done.

Reply:
Thanks.